README.md

```diff
@@ -123,6 +123,7 @@ External dependencies:
  - libfuse-dev          >= 2.8  LGPL v2.1       https://github.com/libfuse/libfuse
  - libre2-dev           >= 2013 BSD 3-clause    https://github.com/google/re2
  - libncurses5-dev      >= 5.7  MIT             https://www.gnu.org/software/ncurses/
```

Comment:
I kind of like the way the name and version collapse, but the license and URL would indeed be nice to have in separate columns, so on the whole I'd be in favour of table-izing them.

Reply:
Sounds good. I decided that I'll throw the formatting into another PR so as not to distract or gate this one in case we end up discussing the formatting more.